0.1.Copy data from Excel codes.R

```diff
@@ -1,79 +1,75 @@
-rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
-gc() #free up memrory and report the memory usage.
+#rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
+#gc() #free up memrory and report the memory usage.
 memory.limit(size = 80000000)
 
 library(xlsx)
 library(ggplot2)
 library(ggpmisc)
 library(readxl)
+library(dplyr)
+library(forcats)
+library(bayestestR)
 
+# set data locations in one place
+inputdata <- c(subjects="Subjects with Clamp Infor.xlsx"
+               ,dat0="Erit-2 Clamp Calculations Dr M with lean mass (003)nm updated with 206 215_2.xlsx");
 
 ## Extracting data from clamp sheet
-subject_infor <- read_excel("E:/2021-03-26 Bulk Fat RNAseq/Subjects with Clamp Infor.xlsx",
+subject_infor <- read_excel(inputdata['subjects'],
                             sheet = "Subjects.infor", col_types = c("numeric", "text"))
-col.location <- read_excel("E:/2021-03-26 Bulk Fat RNAseq/Subjects with Clamp Infor.xlsx",
+col.location <- read_excel(inputdata['subjects'],
                             sheet = "col.location")
 
-for ( i in 1:nrow(subject_infor)) {
-  i=i
-  x <- read_excel("E:/2021-03-26 Bulk Fat RNAseq/Erit-2 Clamp Calculations Dr M with lean mass (003)nm updated with 206 215_2.xlsx", 
-                  sheet = subject_infor$Sheet_Name[i], col_names = FALSE)

```

Comment:
In lines 19 and 20 I think is the cause of the numeric columns getting converted to text. You're reading the entire sheet and later on extracting a range from it. But because the whole sheet contains a mix of data types, it is already converted to `character` by the time you subset the needed rows and columns.